live/app/templates/index.hbs

```diff
@@ -13,17 +13,21 @@
   </section>
 
   {{! I don't know how to correctly test this with our current Mirage config/use :-( }}
-    <section class="index-page__section index-page__section--challenges index-page-challenges">
-      <div class="index-page-challenges__container">
-        <div class="index-page-challenges__presentation">
-          <h2 class="index-page-challenges__presentation-title">Le défi <span class="text--marigold">Pix</span> de la semaine</h2>
-          <p class="index-page-challenges__presentation-text">Chaque semaine, testez vos compétences numériques sur un nouveau sujet.</p>
-        </div>
-        <div class="index-page-challenges__course-list">
-          {{course-list courses=model.coursesOfTheWeek startCourse='startCourse' limit=2}}
-        </div>
+  <section class="index-page__section index-page__section--challenges index-page-challenges">
+    <div class="index-page-challenges__container">
+      <div class="index-page-challenges__presentation">
+        <h2 class="index-page-challenges__presentation-title">Les défis <span class="text--marigold">Pix</span> de la semaine</h2>
+        <p class="index-page-challenges__presentation-text">Chaque semaine, testez vos compétences numériques sur un nouveau sujet.</p>
       </div>
-    </section>
+      <div class="index-page-challenges__course-list">
+        {{course-list courses=model.coursesOfTheWeek startCourse='startCourse' limit=2}}
+      </div>
+    </div>
+    <div class="index-page-challenges__div-link-old-courses-of-the-week">
```

Comment:
renommé en index-page-challenges__courses-of-the-week et index-page-challenges__courses-of-the-week-link